client/features/auth-pages/SignUpPage.tsx

```diff
@@ -90,7 +111,14 @@ const SignUpPage = () => {
   };
 
   const onSubmit: SubmitHandler<FormData> = async (data) => {
-    const { email, password, name } = data;
+    const { email, password, name, confirmPassword } = data;
+
+    if (password !== confirmPassword) {
```

Comment:
Since we are using the error texts on the react hook form to display error messages, we actually don't need to snackbar.

Reply:
Thank you, just made changes and pushed to my branch